### ISSUE TITLE
Making gifs with animations?

### ISSUE THREAD

**Issue by ivanshiny** [gdkchan/SPICA#54]
Hi, I'm trying to get some models into gifs (for example, a pokemon doing an attack animation), but I don't know how to do so, and since the "save" option only allows to export as .dae, .smd and .bch.

Is there any way for me to create a gif using those files???

Thanks in advance.

**Comment by legoj15** [comment]
Not with this program, no. You'll have to export the models, textures, and animations and import them into a animation software to render it.

**Comment by ivanshiny** [comment]
Oh OK, do you know any software I can use?

**Comment by legoj15** [comment]
Blender has native support for .dae files, and you can install a plugin that lets you import smd animations. It's also free.

**Comment by ivanshiny** [comment]
I tried blender with the dae files, but it didn't load the texture, only the white model

**Comment by legoj15** [comment]
yeah, that's where it gets trickier.... the only thing I can tell you is that what you're trying to do, I've done in blender, but it requires experience. the most I can tell you to do is search up "how to add image textures in blender"

**Comment by waielal** [comment]
The "white model" results from not setting the specular hardness by the exporter. Blender defaults back to the value -1 which is an invalid value and therefore results in this weird behavior.

**Comment by Wambosa** [comment]
another cheap trick is to just screen record the section where the animation is playing. If you need fanciness like transparency, then this won't work.
